### PR TITLE
feat: buttons

### PR DESCRIPTION
**Pull request by divyakoli1401**
image

### CODE REVIEW

File: buttons.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+@import url(colors.css);
+
+.button-container{
+    display: flex;
+    flex-direction: row;
```
Comment (MaruthiPothuganti) [inline buttons.css]: Suggestion: can use a short hand property `flex-flow: row wrap;`

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended: use of short hand property

File: buttons.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+/* link button */
+
+.btn-link a{
+text-decoration: none;
+color: var(--font-color-dark);
+}
+
+
```
Comment (tanishq20) [inline buttons.css:31]: Avoid giving these extra spaces.

File: buttons.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+.button-container{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 2rem;
+    align-items: center;
+}
+
+button{
```
Comment (tanishq20) [inline buttons.css]: You can use `btn` class to give these properties. In the future, if you don't want these properties in any button then simply remove `btn` class from the button.

File: buttons.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+/* float action icon */
+
+.btn-float-icon-container{
+    position: relative;
+}
+
+.btn-float-icon{
+ position: absolute;
+ z-index: 999;
```
Comment (tanishq20) [inline buttons.css]: A high z-index value may cause problems in future codes.

File: buttons.css [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+ bottom: 20px;
+ right: 20px;
+ background-color: var(--complement-color);
+ box-shadow: 0 6px 5px gray;
+}
+
+.btn-float-icon:hover{
+    background-color: var(--btn-complement-hover);
+
```
Comment (tanishq20) [inline buttons.css:51]: Avoid giving these spaces.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+<body>
+    <div class="button-container">
+        <button class="btn-primary">Primary</button>
+        <button class="btn-secondary">Secondary</button>
+        <button class="btn-success">Success</button>
+        <button class="btn-danger">Danger</button>
+        <button class="btn-link">
+            <a href="#">Link</a>
+        </button>
+        <button class="btn-icon fa fa-comment fa-2x"></button>
```
Comment (tanishq20) [inline index.html]: 2x Icon seems too big. I feel a small icon will look better.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+        <button class="btn-icon fa fa-bell fa-2x"></button>
```
Comment (tanishq20) [inline index.html]: `fa fa-bell fa-2x` instead of this you can give `<i class="fa fa-bell fa-2x"></i>` inside button tag.

File: buttons.css (hunk 3 above)
Comment (divyakoli1401) [reply buttons.css]: Hi tanishq, using button element selector serves the same purpose.

File: buttons.css (hunk 4 above)
Comment (divyakoli1401) [reply buttons.css]: float action icon is supposed to be on top of the page in every website

File: buttons.css (hunk 5 above)
Comment (divyakoli1401) [reply buttons.css:51]: will take care of it

File: index.html (hunk 6 above)
Comment (divyakoli1401) [reply index.html]: they would look too small on a desktop screen

File: index.html (hunk 7 above)
Comment (divyakoli1401) [reply index.html]: thanks for the suggestion tanishq

File: buttons.css (hunk 1 above)
Comment (divyakoli1401) [reply buttons.css]: made the changes thank you maruthi

File: buttons.css (hunk 2 above)
Comment (divyakoli1401) [reply buttons.css:31]: will take care of it tanishq thank you

**Pull request merged**